profiles/apparmor.d/usr.share.openqa.script.worker

```diff
@@ -145,6 +150,8 @@
   /var/lib/openqa/share/factory/tmp/ rw,
   /var/lib/openqa/share/factory/tmp/** rw,
   /var/lib/openqa/share/tests/** r,
+  /var/lib/openqa/share/tests/*/needles/** rw,
+  /var/lib/openqa/share/tests/*/profiles/*/needles/** rw,
```

Comment:
Is it worker process that writes them? I'm pretty sure it worked for me before - without this change.

Reply:
No, the worker should not write anything in share. Wer have many workers setup mounting share as readonly over NFS

Reply:
Yes, I agree -- I guess I pasted that into the wrong file at some point when I meant to put it in the openqa one to allow needle editing via the webUI. Sorry about that.

 Should I revert that bit of the patch as a new PR?

Reply:
I would appreciate that, yes